index.css

```diff
@@ -7,24 +7,90 @@
   --secondary: #f4f6fd;
 
   --blue: #1c1f6e;
-  --green: #ff6d6d;
-  --red: #14cdc8;
+  --shadow: #1c1f6e1f;
+  --red: #ff6d6d;
+  --green: #14cdc8;
   --yellow: #f9d100;
   --white: #ffffff;
 }
 
+html {
+  font-size: 62.5%;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 *,
 *::before,
 *::after {
   box-sizing: border-box;
   border: none;
+  margin: 0;
+  padding: 0;
+}
+a {
+  text-decoration: none;
+  color: white;
 }
 
 img {
   display: block;
   max-width: 100%;
+  margin: 0.5rem;
+}
+body {

```

Comment:
Add an empty line above the `body` tab.  Let's make sure the format and spacing are consistent and readable.